tests/checkpoint/test_checkpoint.py

```diff
@@ -921,6 +929,25 @@ def test_checkpoint_run_adds_requisite_ids(self, tmp_path: pathlib.Path):
         assert meta["checkpoint_id"] == checkpoint.id
         assert meta["validation_id"] == checkpoint.validation_definitions[0].id
 
+    @pytest.mark.filesystem
+    def test_checkpoint_run_with_data_docs_and_slack_actions_emit_page_links(
+        self, tmp_path: pathlib.Path
+    ):
+        actions = [
+            UpdateDataDocsAction(name="docs_action"),
+            SlackNotificationAction(name="slack_action", slack_webhook="webhook"),
```

Comment:
Is there a test with a `SlackNotificationAction` and no `UpdateDataDocsAction`? That is, do we test that we can run a checkpoint with and without a data docs action when combined with another action?

Also, would it be useful to do this test with the actions sorted in the opposite order. I know we always run the data docs action first and we have some testing. I don't know if we have an e2e test for that.

Reply:
Flipped the order and added a new test with just slack